src/fhir/testUtilities.tsx

```diff
@@ -0,0 +1,123 @@
+import { Bundle } from "@smile-cdr/fhirts/dist/FHIR-R4/classes/bundle";
+import { getErrors } from "./api";
+import { FhirContext } from "../components/fhir/FhirContext";
+import ReportForm from "../components/reports/ReportForm";
+import { noValues } from "../components/reports/FormDefaults";
+import FHIR from "fhirclient/lib/entry/browser";
+import React from "react";
+
+/**
+ * Utilities to be used only during testing, no actual tests here.
+ */
+
+const FHIR_URL = process.env.REACT_APP_FHIR_URL || "";
+
+export const createPractitioner = async (practitioner: any) => {
+  const sendPractitioner = await fetch(`${FHIR_URL}/Practitioner`, {
+    method: "POST",
+    body: JSON.stringify(practitioner),
+    headers: {
+      "Content-Type": "application/json",
+    },
+  });
+  await new Promise((r) => setTimeout(r, 500));
+  return checkResponseOK(sendPractitioner);
+};
+
+export const sendBundle = async (bundle: Bundle) => {
+  const sentBundle = await fetch(`${FHIR_URL}/`, {
+    method: "POST",
+    body: JSON.stringify(bundle),
+    headers: {
+      "Content-Type": "application/json",
+    },
+  });
+  await new Promise((r) => setTimeout(r, 500));
+  return checkResponseOK(sentBundle);
+};
+
+const checkResponseOK = async (response: Response) => {
+  const r = await response.json();
+  if (!response.ok) {
+    console.error(JSON.stringify(r));
+    throw new Error(response.statusText);
+  }
+  if (!(r.type === "bundle-response")) {
+    return r;
+  }
+  const errors = getErrors(r);
+  if (errors.length > 1) {
+    errors.forEach((error) => {
+      const message = error.diagnostics;
+      throw new Error(message);
+    });
+  }
+
+  console.debug(`check response: ${JSON.stringify(r, null, 2)}`);
+  return r;
+};
+
+export const getResources = async (resource: string, id?: string): Promise<Bundle> => {
```

Comment:
if we're going to do this, we should constrain the `resource` to be  a type which can either have the values "Practitioner" | "Patient"

Reply:
and "Observation" (for variant tests)